core/broadcast.go

```diff
@@ -237,12 +237,12 @@ type broadcastPacket = *drand.DKGPacket
 const maxQueueSize = 1000
 
 // senderQueueSize returns a dynamic queue size depending on the number of nodes
-// to contact.
+// to contact multiplied by the number of steps in the DKG.
 func senderQueueSize(nodes int) int {
-	if nodes > maxQueueSize {
+	if 3*nodes > maxQueueSize {
 		return maxQueueSize
 	}
-	return nodes
+	return 3 * nodes //nolint:gomnd
```

Comment:
dunno if this is overkill, but `int(FinishPhase) - 2` avoid us having to update this if the DKG gets updated.  Maybe the DKG will never get updated though